packages/@aws-cdk/aws-stepfunctions-tasks/lib/stepfunctions/start-execution.ts

```diff
@@ -30,6 +30,15 @@ export interface StepFunctionsStartExecutionProps extends sfn.TaskStateBaseProps
    * @default - None
    */
   readonly name?: string;
+
+  /**
+   * Pass the execution ID from the context object to the execution input.
+   *
+   * @see https://docs.aws.amazon.com/step-functions/latest/dg/concepts-nested-workflows.html#nested-execution-startid
+   *
+   * @default - false
```

Comment:
@BenChaimberg you mention [here](https://github.com/aws/aws-cdk/issues/14778#issuecomment-865331894) that this should default to `true` however that would result in a breaking change (for people with `input` that is not an object) and this is a stable module.